CHANGES.rst

```diff
@@ -36,6 +36,9 @@ New Features:
   sharing UI widget or other user/groups search widgets.
   [sneridagh]
 
+- Add ``available_roles`` attribute to the GET @sharing endpoint.
```

Comment:
@sneridagh I wonder if we want to include this in our changelog. We add the sharing endpoint in the same release. :)